File: 03. Programming - C#/TestPreparation3Example/Teacher.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TestPreparation3Example
{
    class Teacher:Person
    {
        private int salary;
        public int Salary
        {
            get
            {
                return salary;
            }
            set
            {
                salary = value;
            }
        }

        public Teacher(string name, string egn,string gender,int salary)
            : base(name, egn, gender)
        {
            this.salary = salary;
        }
    }
}

File: 02. Programming - Java/Lesson9/src/uni/fmi/bachelors/Teacher.java
package uni.fmi.bachelors;

public class Teacher extends Person {
	
	private int salary;

	public Teacher(String name, String gender, String EGN, int salary) {
		super(name, gender, EGN);
		this.salary=salary;
	}

	public int getSalary() {
		return salary;
	}

	public void setSalary(int salary) {
		this.salary = salary;
	}
	
	

}
